<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>684</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1030</width>
    <height>680</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>北京世园会游览导航</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/icon/beijing.png</normaloff>:/icon/icon/beijing.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1030</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="11,0">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>580</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,2,0,2,32">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout4" stretch="7,9,9">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="labelStart">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>起点：</string>
          </property>
          <property name="buddy">
           <cstring>comboBoxStartLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxStartLine">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <item>
           <property name="text">
            <string>选择路线</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxStartSpot">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <item>
           <property name="text">
            <string>选择景点</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout5" stretch="7,9,9">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="labelDestination">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>终点：</string>
          </property>
          <property name="buddy">
           <cstring>comboBoxDstLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDstLine">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <item>
           <property name="text">
            <string>选择路线</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDstSpot">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <item>
           <property name="text">
            <string>选择景点</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout6" stretch="0,19">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonTransfer">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>START</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>30</height>
           </size>
          </property>
          <property name="title">
           <string>游览路线</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="radioButtonMinTime">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>15</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
              </font>
             </property>
             <property name="text">
              <string>景点时间最短路径</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonUparrawTransfer_1">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>15</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
              </font>
             </property>
             <property name="text">
              <string>景点侯选路径_1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonUparrawTransfer_2">
             <property name="text">
              <string>景点候选路径_2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonUparrawTransfer_3">
             <property name="text">
              <string>景点候选路径_3</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelTransferRoute">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>游览路线：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowserRoute">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>动态添加</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icon/icon/add.png</normaloff>:/icon/icon/add.png</iconset>
     </property>
     <addaction name="actionAddAll"/>
     <addaction name="separator"/>
     <addaction name="actionAddLine"/>
     <addaction name="actionAddSpot"/>
     <addaction name="actionAddConnect"/>
     <addaction name="separator"/>
     <addaction name="actionAddByText"/>
    </widget>
    <addaction name="menuAdd"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuWatch">
    <property name="title">
     <string>查看(&amp;W)</string>
    </property>
    <addaction name="actiontoolBar"/>
    <addaction name="actionstatusBar"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
    <addaction name="toolEnlarge"/>
    <addaction name="toolShrink"/>
    <addaction name="separator"/>
    <addaction name="actionLineMap"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionuseHelp"/>
   </widget>
   <widget class="QMenu" name="menu_A">
    <property name="title">
     <string>关于(&amp;A)</string>
    </property>
    <addaction name="actionAuthor"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuTool"/>
   <addaction name="menuView"/>
   <addaction name="menuWatch"/>
   <addaction name="menu_H"/>
   <addaction name="menu_A"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="toolEnlarge"/>
   <addaction name="toolShrink"/>
   <addaction name="separator"/>
   <addaction name="actionAddAll"/>
   <addaction name="actionAddLine"/>
   <addaction name="actionAddSpot"/>
   <addaction name="actionAddConnect"/>
   <addaction name="actionAddByText"/>
   <addaction name="separator"/>
   <addaction name="actionTransfer"/>
   <addaction name="actionLineMap"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionClose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="toolEnlarge">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/enlarge.png</normaloff>:/icon/icon/enlarge.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>点击放大</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="toolShrink">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/shrink.png</normaloff>:/icon/icon/shrink.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>点击缩小</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionTransfer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/query.png</normaloff>:/icon/icon/query.png</iconset>
   </property>
   <property name="text">
    <string>路线选择(右栏)</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionQueryInfo">
   <property name="text">
    <string>信息查询(预留)</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
  <action name="actionAddAll">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/all.png</normaloff>:/icon/icon/all.png</iconset>
   </property>
   <property name="text">
    <string>所有(&amp;A)</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAddLine">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/graden.png</normaloff>:/icon/icon/graden.png</iconset>
   </property>
   <property name="text">
    <string>路线(&amp;L)</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionAddSpot">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/spot.png</normaloff>:/icon/icon/spot.png</iconset>
   </property>
   <property name="text">
    <string>景点(&amp;S)</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionAddConnect">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/connect.png</normaloff>:/icon/icon/connect.png</iconset>
   </property>
   <property name="text">
    <string>连接(&amp;C)</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionAddByText">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/text.png</normaloff>:/icon/icon/text.png</iconset>
   </property>
   <property name="text">
    <string>文本方式(&amp;T)</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actiontoolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>toolBar</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
  <action name="actionstatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>statusBar</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>7</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionLineMap">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/map.png</normaloff>:/icon/icon/map.png</iconset>
   </property>
   <property name="text">
    <string>世园会图</string>
   </property>
   <property name="toolTip">
    <string>世园会网络路线图</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionWheel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/mouse.png</normaloff>:/icon/icon/mouse.png</iconset>
   </property>
   <property name="text">
    <string>鼠标可缩放</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
  <action name="actionQt">
   <property name="icon">
    <iconset theme="Qt" resource="resource.qrc">
     <normaloff>:/icon/icon/qt.png</normaloff>:/icon/icon/qt.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/beijing.png</normaloff>:/icon/icon/beijing.png</iconset>
   </property>
   <property name="text">
    <string>制作者</string>
   </property>
  </action>
  <action name="actionuseHelp">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/help.png</normaloff>:/icon/icon/help.png</iconset>
   </property>
   <property name="text">
    <string>使用帮助</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
